src/app/loginComponents/accounts/accounts.component.ts

```diff
@@ -1,12 +1,53 @@
 import { Component, OnInit } from '@angular/core';
-import { Subject } from 'rxjs';
+import { Location } from '@angular/common';
+import { MatTabChangeEvent } from '@angular/material';
+import { FormControl } from '@angular/forms';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-accounts',
   templateUrl: './accounts.component.html'
 })
 export class AccountsComponent implements OnInit {
+  public currentTab = 'accounts'; // default to the 'accounts' tab
+  protected selected = new FormControl(0);
```

Comment:
This seems a little mysterious to me... I don't know this part of Angular well enough to know if it is the best way.

Can you just let `selected` get set via parseUrl method? Or are you setting it because it can't be null in the accounts.component.html, on line 11?

Reply:
I was just following along the example of the tabs on workflow pages, but it works just the same without being set to 0. Will take that out.